GEMS_TO_RELEASE

```diff
@@ -1,5 +1,5 @@
 {
     "management": [
-        "azure_mgmt_storage"
+        "azure_mgmt_msi"
```

Comment:
Just a question, is this expected?

Reply:
correct, I'm thinking I'll release this gem tomorrow, then do the updates to compute, since this one seems independent, right?

Reply:
Yep, it is independent